diff --git a/node_modules/@ethersproject/base64/LICENSE.md b/node_modules/@ethersproject/base64/LICENSE.md
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/README.md b/node_modules/@ethersproject/base64/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/_version.d.ts b/node_modules/@ethersproject/base64/lib.esm/_version.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/_version.d.ts.map b/node_modules/@ethersproject/base64/lib.esm/_version.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/_version.js b/node_modules/@ethersproject/base64/lib.esm/_version.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/_version.js.map b/node_modules/@ethersproject/base64/lib.esm/_version.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/base64.d.ts b/node_modules/@ethersproject/base64/lib.esm/base64.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/base64.d.ts.map b/node_modules/@ethersproject/base64/lib.esm/base64.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/base64.js b/node_modules/@ethersproject/base64/lib.esm/base64.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/base64.js.map b/node_modules/@ethersproject/base64/lib.esm/base64.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/index.d.ts b/node_modules/@ethersproject/base64/lib.esm/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/index.d.ts.map b/node_modules/@ethersproject/base64/lib.esm/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/index.js b/node_modules/@ethersproject/base64/lib.esm/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib.esm/index.js.map b/node_modules/@ethersproject/base64/lib.esm/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/_version.d.ts b/node_modules/@ethersproject/base64/lib/_version.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/_version.d.ts.map b/node_modules/@ethersproject/base64/lib/_version.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/_version.js b/node_modules/@ethersproject/base64/lib/_version.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/_version.js.map b/node_modules/@ethersproject/base64/lib/_version.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/base64.d.ts b/node_modules/@ethersproject/base64/lib/base64.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/base64.d.ts.map b/node_modules/@ethersproject/base64/lib/base64.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/base64.js b/node_modules/@ethersproject/base64/lib/base64.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/base64.js.map b/node_modules/@ethersproject/base64/lib/base64.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/browser-base64.d.ts b/node_modules/@ethersproject/base64/lib/browser-base64.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/browser-base64.d.ts.map b/node_modules/@ethersproject/base64/lib/browser-base64.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/browser-base64.js b/node_modules/@ethersproject/base64/lib/browser-base64.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/browser-base64.js.map b/node_modules/@ethersproject/base64/lib/browser-base64.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/index.d.ts b/node_modules/@ethersproject/base64/lib/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/index.d.ts.map b/node_modules/@ethersproject/base64/lib/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/index.js b/node_modules/@ethersproject/base64/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/lib/index.js.map b/node_modules/@ethersproject/base64/lib/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/base64/package.json b/node_modules/@ethersproject/base64/package.json
index f9fcb39..f96fbea 100644
--- a/node_modules/@ethersproject/base64/package.json
+++ b/node_modules/@ethersproject/base64/package.json
@@ -3,9 +3,6 @@
     "base64.js": "browser-base64.js"
   },
   "author": "Richard Moore <me@ricmoo.com>",
-  "browser": {
-    "./lib/base64": "./lib/browser-base64.js"
-  },
   "dependencies": {
     "@ethersproject/bytes": "^5.8.0"
   },
