diff --git a/node_modules/@ethersproject/random/LICENSE.md b/node_modules/@ethersproject/random/LICENSE.md
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/README.md b/node_modules/@ethersproject/random/README.md
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/_version.d.ts b/node_modules/@ethersproject/random/lib.esm/_version.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/_version.d.ts.map b/node_modules/@ethersproject/random/lib.esm/_version.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/_version.js b/node_modules/@ethersproject/random/lib.esm/_version.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/_version.js.map b/node_modules/@ethersproject/random/lib.esm/_version.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/index.d.ts b/node_modules/@ethersproject/random/lib.esm/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/index.d.ts.map b/node_modules/@ethersproject/random/lib.esm/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/index.js b/node_modules/@ethersproject/random/lib.esm/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/index.js.map b/node_modules/@ethersproject/random/lib.esm/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/random.d.ts b/node_modules/@ethersproject/random/lib.esm/random.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/random.d.ts.map b/node_modules/@ethersproject/random/lib.esm/random.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/random.js b/node_modules/@ethersproject/random/lib.esm/random.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/random.js.map b/node_modules/@ethersproject/random/lib.esm/random.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/shuffle.d.ts b/node_modules/@ethersproject/random/lib.esm/shuffle.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/shuffle.d.ts.map b/node_modules/@ethersproject/random/lib.esm/shuffle.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/shuffle.js b/node_modules/@ethersproject/random/lib.esm/shuffle.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib.esm/shuffle.js.map b/node_modules/@ethersproject/random/lib.esm/shuffle.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/_version.d.ts b/node_modules/@ethersproject/random/lib/_version.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/_version.d.ts.map b/node_modules/@ethersproject/random/lib/_version.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/_version.js b/node_modules/@ethersproject/random/lib/_version.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/_version.js.map b/node_modules/@ethersproject/random/lib/_version.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/browser-random.d.ts b/node_modules/@ethersproject/random/lib/browser-random.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/browser-random.d.ts.map b/node_modules/@ethersproject/random/lib/browser-random.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/browser-random.js b/node_modules/@ethersproject/random/lib/browser-random.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/browser-random.js.map b/node_modules/@ethersproject/random/lib/browser-random.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/index.d.ts b/node_modules/@ethersproject/random/lib/index.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/index.d.ts.map b/node_modules/@ethersproject/random/lib/index.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/index.js b/node_modules/@ethersproject/random/lib/index.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/index.js.map b/node_modules/@ethersproject/random/lib/index.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/random.d.ts b/node_modules/@ethersproject/random/lib/random.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/random.d.ts.map b/node_modules/@ethersproject/random/lib/random.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/random.js b/node_modules/@ethersproject/random/lib/random.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/random.js.map b/node_modules/@ethersproject/random/lib/random.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/shuffle.d.ts b/node_modules/@ethersproject/random/lib/shuffle.d.ts
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/shuffle.d.ts.map b/node_modules/@ethersproject/random/lib/shuffle.d.ts.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/shuffle.js b/node_modules/@ethersproject/random/lib/shuffle.js
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/lib/shuffle.js.map b/node_modules/@ethersproject/random/lib/shuffle.js.map
old mode 100644
new mode 100755
diff --git a/node_modules/@ethersproject/random/package.json b/node_modules/@ethersproject/random/package.json
index e83177e..3507999 100644
--- a/node_modules/@ethersproject/random/package.json
+++ b/node_modules/@ethersproject/random/package.json
@@ -3,9 +3,6 @@
     "random.js": "browser-random.js"
   },
   "author": "Richard Moore <me@ricmoo.com>",
-  "browser": {
-    "./lib/random": "./lib/browser-random.js"
-  },
   "dependencies": {
     "@ethersproject/bytes": "^5.8.0",
     "@ethersproject/logger": "^5.8.0"
